test/functional/apps/management/_import_objects.js

```diff
@@ -71,7 +71,7 @@ export default function ({ getService, getPageObjects }) {
       await PageObjects.settings.importFile(path.join(__dirname, 'exports', '_import_objects_exists.json'), false);
 
       await PageObjects.header.waitUntilLoadingHasFinished();
-      await PageObjects.settings.setImportIndexFieldOption(2);
+      await PageObjects.settings.associateIndexPattern('d1e4c910-a2e6-11e7-bb30-233be9be6a15', 'logstash-*');
       await PageObjects.settings.clickConfirmChanges();
```

Comment:
@rasroh I think this one needs to be;
`await PageObjects.settings.associateIndexPattern('logstash-*', 'logstash-*');`
because  _import_objects_exists.json has;
`"searchSourceJSON": "{\"index\":\"logstash-*\`

And the failing screenshot shows `logstash-*`

Reply:
used the right IndexPattern id.